.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -63,6 +63,7 @@ body:
       options:
         - "Insider Build (xxxxx)"
         - "Windows 11 (22000)"
+        - "November 2021 Update (19044)
         - "May 2021 Update (19043)"
         - "October 2020 Update (19042)"
         - "May 2020 Update (19041)"
```

Comment:
Nit: I'd propose prefacing all of these with `Windows 10` to reduce confusion. Right now, a user could see "November 2021 update" and think "oh right, that Windows 11 update I got".

Reply:
@riverar agreed, and additionally what do you think about including the version names? Those are the names that appear when you run `winver` so possibly more understandable to devs?

```
      options:
        - "Insider Build (xxxxx)"
        - "Windows 11 version 21H2 (22000)"
        - "Windows 10 version 21H2 (November 2021 Update, 19044)
        - "Windows 10 version 21H1 (May 2021 Update, 19043)"
        - "Windows 10 version 20H2 (October 2020 Update, 19042)"
```

Should we just get rid of the "marketing version names" in the list all together?

```
      options:
        - "Insider Build (xxxxx)"
        - "Windows 11 version 21H2 (22000)"
        - "Windows 10 version 21H2 (19044)
        - "Windows 10 version 21H1 (19043)"
        - "Windows 10 version 20H2 (19042)"
```

That'd be more consistent with what Visual Studio displays for developers too

![image](https://user-images.githubusercontent.com/13246069/144686355-4a088449-3c62-47c0-8203-175cdfd583e9.png)

Reply:
I'd keep the marketing names as OS versions surface differently all over the place. Otherwise, agree with your changes :thumbsup: